File: js/soal1.js
/*
Soal        : 1
A           : 5
B           : 9
Suku akhir  : 21
*/

A=5
B=9
suku_akhir=2,
u=A,
s=A;
console.log("Deret Aritmatikanya adalah"); 
for(i=0;i<suku_akhir;i++){                            
    console.log("Angka U awal = "+u+"\t");
    console.log("Angka S awal = "+s+"\t"); 
    u=u+B; 
    s=s+u;    
    console.log("Angka U akhir = "+u+"\t");
    console.log("Angka S akhir = "+s+"\t");  
}
console.log("S = s-u = " + s + " - " +u + " = "+ (s-u)); 
s=s-u;  
console.log("Jumlah Deret Aritmatikanya adalah "+s);

File: X/clang/XRPL_UKL1/soal1.c
/*
Soal        : 1
a           : 5
B           : 9
Suku akhir  : 21
*/

#include <stdio.h>

int main(){
    int a=5,
        B=9,
        suku_akhir=2,
        u=a,
        s=a,
        total=0;
    printf("Deret aritmatikanya adalah \n");
    for(int i=0;i<suku_akhir;i++){
        printf("Angka U awal = %d\n", u);
        printf("Angka S awal = %d\n", s); 
        u+=B;
        s+=u;   
        printf("Angka U awal = %d\n", u);
        printf("Angka S awal = %d\n", s); 
    }
    printf("S = s - u = %d - %d = %d\n", s, u, (s-u));
    total=s-u;
    printf("Jumlah deret aritmatikanya adalah %d\n", s);
}
